src/pages/home/report/ReportFooter.tsx

```diff
@@ -153,6 +158,7 @@ function ReportFooter({
                         />
                     )}
                     {isArchivedRoom && <ArchivedReportFooter report={report} />}
+                    {!isArchivedRoom && blockedFromChat && <BlockedReportFooter />}
```

Comment:
This is causing a crash on main:

```
Unexpected text node: . A text node cannot be a child of a <View>.
```

Reply:
@roryabraham Can you tell me how to reproduce the bug, pls?

Reply:
created https://github.com/Expensify/App/pull/43083 to fix it

Reply:
Just open any report